draft-ietf-mls-architecture.md

```diff
@@ -77,26 +77,13 @@ informative:
        -
           ins: Google
 
-  Loopix:
```

Comment:
I think the goal was to add a new reference for the CAP theorem.  The reference for Loopix should remain.